node/Cargo.toml

```diff
@@ -1,31 +1,38 @@
 [package]
-name = "braidpool-node"
+name = "node"
 version = "0.1.0"
 edition = "2021"
 
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
 [dependencies]
+tokio = { version = "1", features = ["rt", "net", "macros", "rt-multi-thread", "tracing", "io-util"] }
+serde = { version = "^1.0", features = ["derive"] }
+bincode = "1.3"
+tokio-util = { version = "0.7.0", features = ["full"] }
+tokio-stream = { version = "0.1" }
+tracing = "0.1"
+tracing-subscriber = { version = "0.3.1", default-features = false, features = ["fmt", "ansi", "env-filter", "tracing-log"] }
+futures = { version = "0.3.28", features = ["thread-pool"]}
+bytes = "1.4.0"
+flexbuffers = { version = "25.2", features = ["deserialize_human_readable", "serialize_human_readable"] }
+serde_json = "1.0.10"
+sqlite = { version = "0.36" }
+clap = { version = "4.4.2", features = ["derive", "string"] }
+bitcoin = { git = "https://github.com/braidpool/rust-bitcoin.git", branch = "cpunet" }
```

Comment:
We should make a tagged release on rust-bitcoin for this rather than using a branch.